### PR TITLE
Add two notes to Matrix integration

### PR DESCRIPTION
**Pull request by 235816**
## Proposed change

This pull request adds two notes to the documentation of the Matrix integration.

The second note is copied from the documentation of the telegram integration (source/_integrations/telegram.markdown).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @235816, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18229).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/matrix.markdown [hunk 1]
```diff
@@ -190,3 +196,17 @@ action:
       images:
         - /path/to/picture.jpg
 ```
+
+<div class='note'>
+
+Since Home Assistant version 0.48 you have to [whitelist the source folder](/docs/configuration/basic/) of the file you want to include in the notification.
```
Comment (frenck) [inline source/_integrations/matrix.markdown]:
```suggestion
If you need to include a file from an external folder in your notifications, you will have to [whitelist the source folder](/docs/configuration/basic/).
```

**Comment by homeassistant** [comment]
Hi @235816, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18229).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/matrix.markdown [hunk 2]
```diff
@@ -190,3 +196,17 @@ action:
       images:
         - /path/to/picture.jpg
 ```
+
+<div class='note'>
+
+If you need to include a file from an external folder in your notifications, you will have to [whitelist the source folder](/docs/configuration/basic/).
```
Comment (frenck) [inline source/_integrations/matrix.markdown]:
```suggestion
If you need to include a file from an external folder in your notifications, you will have to [list the source folder as allowed](/docs/configuration/basic/).
```

**Comment by homeassistant** [comment]
Hi @235816, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18229).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @235816 👍

**Pull request merged**